test_app/tests/authentication/conftest.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+
+from ansible_base.lib.utils.auth import get_organization_model, get_team_model
+from ansible_base.rbac import permission_registry
+from ansible_base.rbac.models import RoleDefinition
+
+SYSTEM_ROLE_NAME = 'System role'
+TEAM_MEMBER_ROLE_NAME = 'Team Member'
+ORG_MEMBER_ROLE_NAME = 'Organization Member'
+
+
+@pytest.fixture
+def system_role():
+    return RoleDefinition.objects.create(
+        name=SYSTEM_ROLE_NAME,
+    )
+
+
+@pytest.fixture
+def team_member_role():
+    return RoleDefinition.objects.create(
+        name=TEAM_MEMBER_ROLE_NAME,
+        content_type=permission_registry.content_type_model.objects.get_for_model(get_team_model()),
+        managed=True,
+    )
+
+
+@pytest.fixture
+def organization_member_role():
+    return RoleDefinition.objects.create(
+        name=ORG_MEMBER_ROLE_NAME,
+        content_type=permission_registry.content_type_model.objects.get_for_model(get_organization_model()),
+        managed=True,
+    )
```

Comment:
Both of these fixtures already exist in the path where fixtures are available as `member_rd` and `org_member_rd`. I'd prefer to not introduce new fixtures for this purpose. But this is can be touched up later, since it's just a test change.

Reply:
they are in rbac's conftest, and they're expecting they exists. So I would rather spend time on it later

Reply:
> they are in rbac's conftest

That is not the case https://github.com/ansible/django-ansible-base/blob/devel/test_app/tests/conftest.py#L630